terraform/main.tf

```diff
@@ -18,3 +20,39 @@ output "serverless-secret-access-key" {
   value     = module.serverless-user.aws_secret_access_key
   sensitive = true
 }
+
+
+locals {
+  s3_policy = jsonencode({
+    "Version" : "2012-10-17",
+    "Statement" : [
+      {
+        "Effect" : "Allow",
+        "Action" : [
+          "s3:GetBucketPolicy",
+        ],
+        "Resource" : [
+          "arn:aws:s3:::mfa-api-*-serverlessdeploymentbucket*",
+        ]
+      },
+    ]
+  })
+
+  api_gateway_policy = jsonencode({
+    "Version" : "2012-10-17",
+    "Statement" : [
+      {
+        "Effect" : "Allow",
+        "Action" : [
+          "apigateway:UpdateRestApiPolicy",
+        ],
+        "Resource" : [
+          // dev-mfa-api
+          "arn:aws:apigateway:${var.aws_region}:*:restapis/7f2jflg37i",
```

Comment:
This is concerning to me. However we want to handle this situation, I'm really hesitant to hard-code any instance-specific data into our codebase. In theory, anyone else could be running this application as well.